assets/app/view/game/route_selector.rb

```diff
@@ -167,6 +167,27 @@ def render
           ]
         end
 
+        bonuses = @game.revenue_bonuses(current_entity).map do |bonus|
```

Comment:
is this necessary?

Reply:
It's not necessary, but it is very helpful. Here's a screenshot of what this looks like:
![route-selector](https://user-images.githubusercontent.com/11144854/207148066-7e72ca7b-d7bd-4fe8-8323-c51a50363d68.png)
Without the addition of this bonus revenue, the player would just see the 2M and 2H trains in the route selector list, and a total revenue of Pt125. By adding this extra revenue from the private company (C&B in this case) it makes it clearer what the corporation's revenue will be.
I think that this would be useful in other titles too, for example for the QLB's revenue bonus in 1873.

Reply:
Instead of adding to the train row, could the Submit button just be modified with the grand total from the privates? I'm thinking something along the lines of 1822's mail contract bonus - 
![Screenshot 2022-12-12 212739](https://user-images.githubusercontent.com/1045173/207226901-f84a0537-7f21-4efb-8a85-a49162fd845d.png)

Reply:
Yes, that would be easy enough to do (it's just overriding the `submit_revenue_str` method). I thought that showing the additional revenue sources in the table made it even clearer what has happening, but it's not necessary.

Reply:
This has been done, removing this change.